lib/bundles/inspec-compliance/api.rb

```diff
@@ -151,12 +178,23 @@ def self.get_token(config)
 
     def self.target_url(config, profile)
       if config['server_type'] == 'automate'
-        target = "#{config['server']}/#{profile}/tar"
+        owner, id = profile.split('/')
+        target = "#{config['server']}/profiles/#{owner}/#{id}/tar"
       else
         owner, id = profile.split('/')
         target = "#{config['server']}/owners/#{owner}/compliance/#{id}/tar"
       end
       target
     end
+
+    # returns a parsed url for `admin/profile` or `compliance://admin/profile`
+    def self.sanitize_profile_name(profile)
```

Comment:
I just learned yesterday that Automate is using fancy usernames like `adam@leff.com`, which breaks the current parsing since `adam@` is not part of host. I agree that we need way more testing around this